Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/coronatest/ui/CoronaTestTestFragment.kt

```diff
@@ -65,26 +65,30 @@ class CoronaTestTestFragment : Fragment(R.layout.fragment_test_coronatest), Auto
             qrcodeScanViewfinder.setCameraPreview(binding.qrcodeScanPreview)
         }
 
-        vm.pcrtState.observe2(this) {
+        viewModel.pcrtState.observe2(this) {
```

Comment:
vm -> viewModel
how is the naming in other classes?

Reply:
usually, we call it `viewModel` in a new code, so I used this opportunity for a small refactoring